source/github-helpers/dom-formatters.tsx

```diff
@@ -14,7 +14,8 @@ export const linkifiedURLClass = 'rgh-linkified-code';
 export const linkifiedURLSelector = '.rgh-linkified-code';
 
 export const codeElementsSelector = [
-	'.blob-code-inner', // Code lines
+	// Sometimes formatted diffs are loaded later and discard our formatting #5870
+	'.blob-code-inner:not(deferred-diff-lines.awaiting-highlight *)', // Code lines
```

Comment:
Here's the HTML diff between the uncached and cached first loads:

<img width="964" alt="Screenshot 4" src="https://github.com/refined-github/refined-github/assets/1402241/330ab85b-8d2c-4361-bd61-562a6f819ffc">